pkg/usage/usage.go

```diff
@@ -149,8 +154,13 @@ func (u *Usage) SetApplicationID(appID string) *Usage {
 }
 
 // SetApplicationVersion : usecase(project-version)
-func (u *Usage) SetApplicationVersion(appVersion string) *Usage {
-	u.appVersion = appVersion
+func (u *Usage) SetApplicationVersion(appVersion, gitcommit string) *Usage {
+	if gitcommit == "" {
+		u.appVersion = appVersion
+		return u
+	}
+
+	u.appVersion = strings.Join([]string{appVersion, gitcommit[0:7]}, "-")
```

Comment:
(optional) This might look simpler, if strings.Join does not offer any kind of nil safety operations,  considering appVersion cannot be nil it has to be `""` at worst.

```suggestion
	u.appVersion = appVersion + "-" + gitcommit[0:7]
```

Reply:
yes, we can convert it since there are two args.